maxminddb/reader.py

```diff
@@ -152,6 +166,35 @@ def get_with_prefix_len(
             return self._resolve_data_pointer(pointer), prefix_len
         return None, prefix_len
 
+    def __iter__(self, include_aliased_nodes=False):
```

Comment:
It looks like the C version doesn't accept this parameter. Does that matter?

Reply:
Oh, I meant to drop support for this. I'll remove it. I am not sure anyone actually wants the aliased notes and figured I'd leave support for them to a follow-up story.